test/ServiceBus.Tests/Streaming/EHSlowConsumingTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Orleans.Runtime.Configuration;
+using Orleans.ServiceBus.Providers;
+using Orleans.Storage;
+using Orleans.Streams;
+using Orleans.TestingHost;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestExtensions;
+using Xunit;
+
+namespace ServiceBus.Tests.Streaming
+{
+    [TestCategory("EventHub"), TestCategory("Streaming")]
+    [Collection(TestEnvironmentFixture.DefaultCollection)]
+    class EHSlowConsumingTests : OrleansTestingBase, IClassFixture<EHSlowConsumingTests.Fixture>
```

Comment:
Wow I didn't notice this ! there's another EHSlowConsumingTests.cs  on the top... That's the real one. I will delete this one. Thanks for noticing